src/main/java/com/botdarr/clients/discord/DiscordResponseBuilder.java

```diff
@@ -364,7 +364,7 @@ private DiscordResponse getListOfCommands(List<Command> commands) {
     EmbedBuilder embedBuilder = new EmbedBuilder();
     embedBuilder.setTitle("Commands");
     for (Command com : commands) {
-      embedBuilder.addField(CommandContext.getConfig().getPrefix().replace(" ", "-") + com.getCommandUsage(), com.getDescription(), false);
+      embedBuilder.addField(CommandContext.getConfig().getPrefix() + com.getCommandUsageWithInfix("-"), com.getDescription(), false);
```

Comment:
I'm about a bit confused about this change? Why is it necessary (is discord integration broken)?

Reply:
My apologies, should have created an issue with more explanation first. You can decline it np.

The dashes were missing in the /help messages, make a quick fix for a friend who asked me.

Before:
![image](https://user-images.githubusercontent.com/127911448/225443714-77bac062-86e7-4f0b-87a4-eb743e3061ce.png)

After:
![image](https://user-images.githubusercontent.com/127911448/225444530-9a073115-58c7-4ed5-bbf0-5bc550d00204.png)

Reply:
yea you are def right, the help commands are not correct in discord, however it shouldn't really matter for slash commands (unless you are not using them, but based on that screenshot you must be cause of the "/" prefix)

lemme figure out where this change should really live (likely in the DiscordResponseBuilder)

Reply:
yea its goofy because discord doesn't support spaces in their slash commands, so I used "-", but then when I fix it breaks non slash commands

need to think about best way to fix this cause I think right now the display for help on slash commands is kinda weird

like "/movie-id-add-movie-title-movie-tmdbid" is not actually what you type technically

you would type "/movie-id" then it would auto fill and let you enter the choices

Reply:
ok I did something like this for slash commands
![image](https://user-images.githubusercontent.com/5449086/225806820-503246dc-ff6f-4c39-8504-f3f469463f6c.png)

and something like this for none slash commands
![image](https://user-images.githubusercontent.com/5449086/225806857-c1a121d8-0eed-4ccf-a22f-5bee74ab91c7.png)

if this is good, lemme know, and I'll push on this branch

Reply:
Sorry have been away from the PC for a few days. 

Your updates look great, didn't even realize the non slash comments were different.

Reply:
fixing this here https://github.com/shayaantx/botdarr/pull/94